### DISCUSSION TITLE
Command 'reload vibrancy' resulted in an error

### DISCUSSION THREAD

**Discussion by dawit-michael** [illixion/vscode-vibrancy-continued#44] [Q&A] [answered]
When trying to reload vibrancy  i get an error " ( command 'extension.updateVibrancy' not found)"

**Comment by illixion** [comment, marked as answer]
I’ve just released version 1.1.11 that fixes this, try relaunching VSCode so that it checks for updates.

**Reply by dawit-michael** [reply]
@illixion  apparently running as administrator fixed that issue. thanks .

**Reply by versacesocks** [reply]
How do you run it as an administrator on mac OS ?

**Reply by illixion** [reply]
@versacesocks Why do you need to do that, are you still on 1.1.11?

**Reply by versacesocks** [reply]
No, I'm on 1.1.24. However, as I use the command 'reload vibrancy', it results in an error.

**Reply by illixion** [reply]
What does the error message say?

**Reply by versacesocks** [reply]
Sorry for the late reply. 
<img width="265" alt="Screenshot 2023-11-28 at 10 53 48" src="https://github.com/illixion/vscode-vibrancy-continued/assets/108875587/15d665cf-495b-4f4b-9669-3099ab68e586">

**Reply by illixion** [reply]
Thanks, the FAQ has the solution: https://github.com/illixion/vscode-vibrancy-continued#erofs-read-only-file-system-when-enabling-vibrancy-on-macos

**Reply by versacesocks** [reply]
Hi, I ran the following terminal command and >Reload Vibrancy resulted in an error.

**Reply by illixion** [reply]
Were there any errors when running the command? Have you tried the first step of moving VSCode into your Applications folder, or out and then back in if it’s already there?

**Reply by versacesocks** [reply]
I reinstalled vsc and the command seemed to went through, but I don't know how to activate the transparency. Sorry lol.
